src/engine/BaseEngine.cpp

```diff
@@ -2,6 +2,11 @@
 #include <fstream>
 #include "World.h"
 #include "audio/AudioEngine.h"
+#include "audio/NullAudioEngine.h"
+#include "audio/AudioWorld.h"
+#ifdef RE_USE_SOUND
+#   include "audio/OpenALAudioEngine.h"
```

Comment:
unwanted spaces?

Reply:
No, they are intended to signify that the include is inside the `ifndef`